compiler/ballerina-lang/src/main/java/org/ballerinalang/model/symbols/Symbol.java

```diff
@@ -35,6 +35,8 @@
 
     Name getName();
 
+    Name getOriginalName();
```

Comment:
This is not what we discussed right? IIRC, the suggestion was to change `getName()` to return the original name.

Reply:
I think we have discussed to keep the original-name as a new attribute inside the `ballerinalang.model.symbols.Symbol`. In compiler-api symbol, it will return the original name when we called `getName()` (if there is a one) :thinking: